src/OpenLoco/src/Map/MapGenerator/HeightMap.h

```diff
@@ -1,5 +1,6 @@
 #pragma once
 
+#include <algorithm>
```

Comment:
remove

Reply:
Good call. This is a leftover from before I created the .cpp file.